ReactAndroid/src/main/java/com/facebook/react/views/webview/ReactWebViewManager.java

```diff
@@ -134,7 +135,55 @@ public void onPageStarted(WebView webView, String url, Bitmap favicon) {
     public boolean shouldOverrideUrlLoading(WebView view, String url) {
         if (url.startsWith("http://") || url.startsWith("https://") ||
             url.startsWith("file://")) {
-          return false;
+              boolean shouldBlock = false;
```

Comment:
Why? It's just about not making the check specific to certain URL schemes? Since there's an open issue that can be fixed by this PR, I don't see the point of not doing it.